cmd/app/share/share_test.go

```diff
@@ -0,0 +1,56 @@
+package share
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"numerous.com/cli/internal/app"
+	"numerous.com/cli/internal/appident"
+)
+
+func TestCreate(t *testing.T) {
+	ctx := context.TODO()
+	appSlug := "app-slug"
+	organizationSlug := "organization-slug"
+	ai := appident.AppIdentifier{OrganizationSlug: organizationSlug, AppSlug: appSlug}
+	sharedURL := "https://test-numerous.com/share/123"
+	testErr := errors.New("test error")
+
+	t.Run("it calls app lister with expected arguments", func(t *testing.T) {
+		m := mockAppService{}
+		m.On("ShareApp", ctx, ai).Once().Return(app.ShareAppOutput{SharedURL: &sharedURL}, nil)
+
+		err := shareApp(ctx, &m, Input{AppDir: "", AppSlug: appSlug, OrgSlug: organizationSlug})
+
+		assert.NoError(t, err)
+		m.AssertExpectations(t)
+	})
+
+	t.Run("returns error if empty shared URL returned", func(t *testing.T) {
```

Comment:
Technically I guess a `nil` value is "empty", but I associate more the empty string as an empty value. What do you think?

Reply:
Updated the test name.